tools/eventGenerator.cpp

```diff
@@ -195,7 +192,7 @@ void parseScript( const char* filename, Events& events )
 void printUsageAndExit( const char* name, const int code, bool full = false )
 {
     std::cerr << "Usage: " << name
-              << " [--schema,-s uri] [script]" << std::endl
+              << " [script]" << std::endl
```

Comment:
What is this? Prints:
    Usage: ./eventGenerator [script]
           ./eventGenerator --help

Why not add --help in the Usage?